README.md

```diff
@@ -43,6 +43,8 @@ Sage has excellent performance characteristics (5x faster than - the closed sour
 
 # Installation
 
+## Installing the development version
```

Comment:
I wonder if "Installation" is the proper terminology to use here (I realize I used that as the heading...). Perhaps "Compiling the development version"?

Reply:
I think that even though it is not 100% accurate, it is a good "you know what i mean" expression